iowrappers/redis_client.go

```diff
@@ -65,51 +59,52 @@ func (r *RedisClient) Destroy() {
 func CreateRedisClient(url *url.URL) *RedisClient {
 	password, _ := url.User.Password()
 	return &RedisClient{client: *redis.NewClient(&redis.Options{
-		Addr:     url.Host,
-		Password: password,
+		Addr:         url.Host,
+		Password:     password,
+		PoolSize:     20,              // Max number of socket connections
+		MinIdleConns: 5,               // Minimum number of idle connections to maintain
+		MaxRetries:   3,               // Max number of retries before giving up
+		DialTimeout:  5 * time.Second, // Timeout for establishing new connections
+		ReadTimeout:  3 * time.Second, // Timeout for socket reads
+		WriteTimeout: 3 * time.Second, // Timeout for socket writes
+		PoolTimeout:  4 * time.Second, // Amount of time client waits for connection if all connections are busy
 	})}
```

Comment:
🎉 nice set of options . What is the default `MaxRetries` ? Why not set to `2` and if need to `3` ?